python/api/src/araali/aws.py

```diff
@@ -21,6 +21,38 @@ def cf_rm(name):
 def cf_add_vm():
     print("adding vm")
 
+def cf_launch_fortified_vm(stack_name, email, key_pair):
+    template_url = "https://s3.us-west-1.amazonaws.com/araalinetworks.test/quickstart_vm/AraaliVMQuickstartStack.template.json"
```

Comment:
this one could have been the internal one. If user doesnt specify ami_id, use amazon_linux_latest